website/src/config.ts

```diff
@@ -150,8 +151,8 @@ function getAccessionInputField(): InputField {
 
 function getSubmissionIdInputField(): InputField {
     return {
-        name: SUBMISSION_ID_FIELD,
-        displayName: 'Submission ID',
+        name: SUBMISSION_ID_INPUT_FIELD,
```

Comment:
I wouldn't add the `_INPUT` part to this field name - why did you think of doing that? Other fields are also inputs possibly, and the don't have this.